Client/game_sa/CCamSA.cpp

```diff
@@ -20,8 +20,8 @@ CEntity* CCamSA::GetTargetEntity() const
     CEntitySAInterface* pInterface = m_pInterface->CamTargetEntity;
     if (pInterface)
     {
-        CPools* pPools = pGame->GetPools();
-        return pPools->GetEntity((DWORD*)pInterface);
+        auto pools = pGame->GetPools();
+        return pools->GetEntity((DWORD*)pInterface);
```

Comment:
`return pGame->GetPools()->GetEntity(static_cast<DWORD*>(pInterface));`
if static_cast errors, use c style casting or reinterpret_cast

Reply:
Revert "std::shared_ptr for CPoolsSA (Test)"